api/proto/disperser/disperser.proto

```diff
@@ -150,8 +150,8 @@ enum BlobStatus {
 	// for at least one quorum.
 	INSUFFICIENT_SIGNATURES = 5;
 
-	// CONFIRMING means that the blob has been dispersed to DA nodes and it's waiting for the confirmation onchain
-	CONFIRMING = 6;
+	// DISPERSING means that the blob is currently being dispersed to DA Nodes and being confirmed onchain
+	DISPERSING = 6;
```

Comment:
Just making sure, we haven't deployed the Confirming state yet right?

Reply:
No not yet!